ck+/script/data.js

```diff
@@ -315,6 +321,11 @@ function fetchData() {
 			addPoolInfo(j.encounters[i]);
 		}
 
+		//ADD GENERAL FLAGS TO DATA PARSE
+		const feature_flags = j.feature_flags;
+		game_general_flags.badge_boost = feature_flags.badge_boost;
```

Comment:
This is good for now but in the future we might want to go with a more general case parsing solution, what do you think?

Reply:
I think the parsing is fine, every data.json file will have a series of specific parameters that define how the calc should behave. Documentation of the parsing part should be the first one imo.